topo/node/cptx/cptx.go

```diff
@@ -26,18 +38,132 @@ func New(pb *topopb.Node) (node.Implementation, error) {
 
 type Node struct {
 	pb *topopb.Node
+	cliConn *scraplinetwork.Driver
 }
 
 func (n *Node) Proto() *topopb.Node {
 	return n.pb
 }
 
+// WaitCLIReady attempts to open the transport channel towards a Network OS and perform scrapligo OnOpen actions
+// for a given platform. Retries indefinitely till success.
```

Comment:
So, can we have a re-try limit with some exponential backoff ?

Reply:
Will definitely look into this.

Reply:
@shubh90 I have added a retry limit with exponential backoff part of the 2nd commit in this PR. Do take a look when you get a chance.